lib/takeaway.rb

```diff
@@ -0,0 +1,49 @@
+require_relative 'menu'
+require 'twilio-ruby'
+
+
+class Takeaway
+    
+    attr_accessor :dishes, :orders, :delivery_time, :text
+
+    def initialize(menu = Menu.new)
+        @dishes = menu
+        @orders = []
+    end
+
+    def print_dishes
+        @dishes.display
+    end
+
+    def select(dish)
+        @orders << @dishes.dishes[dish - 1]
+    end
+
+    def total
+        cost = 0 
+        @orders.each do |dish|
+            cost += dish[:price]  
+        end
+        cost
+    end
+
+    def calc_delivery_time
+        Time.now + 60 * 60
+    end
+
+    def send_text
+        account_sid = "asdfgasrerefdav4264655ywgs56uw5y" # TODO Replace this with your own Account SID.
+        auth_token = "wyegse56e3345yeg56eue6u65ey3456" # TODO Replace this with your own Auth Token.
+
+        client = Twilio::REST::Client.new(account_sid, auth_token)
+
+        from = "+55555555555"  # TODO Use the trial number provided by Twilio.
+        to = "+4455555555555"    # TODO Use a verified number to send tests to.
```

Comment:
Presume these are just fakes for github upload purposes? 
Consider using environment variables instead. I found the [precourse material on environment variables](https://makersacademy.teachable.com/courses/256825/lectures/3989201) helpful:

__" Put your secret key in an environment variable and then read it from your code when you need it. For example:

`export SECRET_KEY=12345abcde`

Then, in your Ruby code, read the value:

`secret_key = ENV['SECRET_KEY']`

This way you'll be able to share your code with anyone and upload it to a GitHub repository without compromising your keys."-_